### PR TITLE
Enable `Symbol` polyfill for all versions of Opera Mini.

### LINKED ISSUE #1343: Polyfill Symbol is not served in Opera Mini Browser
According to http://caniuse.com/#search=symbol `Symbol` is not support on any version of Opera Mini.
Additionally, we have verified it on some versions of Opera Mini browser (29.0.2254, 7.6.40234, 7.5.33361, 28.0.2254 and 7.6.35843) with polyifill.io script included (https://cdn.polyfill.io/v2/polyfill.js?features=default&unknown=polyfill)

you can see it's not included when you test it here: https://cdn.polyfill.io/v2/polyfill.js?features=default&unknown=polyfill&ua=Opera/9.80%20(Android;%20Opera%20Mini/7.5.33942/69.190;%20U;%20en)%20Presto/2.12.423%20Version/12.16

It throws this => `Uncaught exception: ReferenceError: Undefined variable: Symbol`.

### PR DESCRIPTION
**Pull request by matzimowski**
http://caniuse.com/#search=symbol

closes https://github.com/Financial-Times/polyfill-service/issues/1343

### CODE REVIEW

**Comment by JakeChampion** [comment]
@matzimowski thanks for the PR! Before we can merge this we would need to know if the polyfills actually work in this browser. Could you run the tests on this browser and post a screenshot showing that they are passing at all?

To run the tests locally you need to:
- clone the project from github
- run `npm install`
- run `npm start`
- visit this url in the browser you are testing with `localhost:3000/test/tests?feature=Symbol&grep=^Symbol(?!\.)`

**Comment by kdzwinel** [comment]
Hey! We tested in Opera Mini 8. Results below:

screen shot 2017-09-27 at 14 38 41
screen shot 2017-09-27 at 14 39 01

**Review by JakeChampion — APPROVED** [review]

**Pull request merged**
